src/components/Button/index.tsx

```diff
@@ -12,28 +12,28 @@ const buttonVariants = cva(
         primary:
           'bg-button-primary border-button-primary text-white ' +
           'hover:bg-button-primary-hover hover:border-button-primary-hover ' +
-          'focus:bg-button-primary-focus focus:border-button-primary-focus ' +
-          'disabled:bg-button-disabled disabled:text-foreground-subtle disabled:border-button-disabled',
+          'focus:bg-button-primary-focus focus:border-button-primary-focus focus:shadow-blue ' +
```

Comment:
I don't think there's a focus shadow, I think it needs to be `focus-visible:shadow-blue`.

Reply:
Also noticed, you haven't changed the hover state?